import_table.py

```diff
@@ -1,88 +1,166 @@
-# ELSE: file is treated as containing leaders 
+# If you need to import more data,
+# modify these dictionaries to include the additional attributes you need.
+PARTICIPANT_DATA_ATTR = {
+    'First Name':'first_name',
+    'Last Name': 'last_name',
+    'Q24': 'university',
+    'Q25': 'classTeam',
+    'Major': 'major',
+    'Grad Year': 'gradYear'
+}
+"""
+A dictionary of attribute names that should be searched for and included as non-essential data
+for the <b>participant form</b>.
+</br>
+The keys of this dictionary are the attribute names used in the CSV file.
+</br>
+The values of this dictionary are the attribute names used elsewhere in the bot code.
+"""
+VOLUNTEER_DATA_ATTR = {
+    'First Name':'first_name',
+    'Last Name': 'last_name',
+    'Company': 'company',
+    'Job Title ': 'jobTitle'
+}
+"""
+A dictionary of attribute names that should be searched for and included as non-essential data
+for the <b>volunteer form</b>.
+</br>
+The keys of this dictionary are the attribute names used in the CSV file.
+</br>
+The values of this dictionary are the attribute names used elsewhere in the bot code.
+"""
 
-FILENAME = "leaders.csv" 
-ATTR_DIS = "Discord ID"
-ATTR_EMAIL = "Email"
+# Variable to keep track of stats for report at the end.
+num_duplicates = 0
+num_error = 0
+num_unfinished = 0
+num_entries = 0
+start_time = time.time()
 
-#FILENAME = "participants.csv" 
-#ATTR_DIS = "What is your Discord username?"
-#ATTR_EMAIL = "Username"
+with open(sys.argv[1], 'r') as csv_file:
+    # Try to open the provided file name.
+    try:
+        reader = csv.DictReader(csv_file, delimiter=',')
+    except:
+        raise BaseException('Cannot read/import CSV file. Check format and resubmit.')
 
-with open(FILENAME, 'r') as csv_file:
-    #Attempt to Read file
+    # Verify that the file has all the attributes we need.
+    attributes = set(reader.fieldnames)
     try:
-        reader = csv.DictReader(csv_file, delimiter=",")
+        attributes.remove('Progress')
+        attributes.remove('Email')
     except:
-        raise BaseException("Cannot read/import csv file. Check format and resubmit")
-    
-    #Search through file for relevant user data
-    userNum = 0
-    for user in reader:
-        userData = {}
-        userData['role'] = []
-        
-        #Gather email and discord_username
-        for attr in user:
-
-            #Find Discord Username
-            if(attr == ATTR_DIS and user[attr] != ""):
-                userData['discord_username'] = user[attr].replace(" ","")
-
-            #Find Email Address
-            if(attr == ATTR_EMAIL and user[attr] != ""):
-                userData['email'] = user[attr].replace(" ", "")
-
-            #Find Role
-            if('participant' in FILENAME.lower()):
-                userData['role'].append('participant')
+        raise ValueError('CSV file missing required attributes. Check file contents and resubmit.')
+
```

Comment:
I determine whether we are importing the participant or the volunteer form based on whether or not the CSV file has a "Roles" attribute. This is assuming that there will never be a scenario where we will have multiple roles for participants, which is a bold assumption to make.

And of course, the relevant attribute in the volunteer form needs to be named "Roles".